docs/developer-guide/map-query-parameters.md

```diff
@@ -0,0 +1,77 @@
+# Map query parameters
+
+In this section we will describe the available map query parameters
```

Comment:
I would name these parameters MapViewer parameters, but this is just my opinion.